files/config/config.yaml

```diff
@@ -1,3 +1,19 @@
 App:
   Host: "http://127.0.0.1"
-  Port: 8000
\ No newline at end of file
+  Port: 8000
+
+DB:
+  DSN: "postgres://go_template:root@127.0.0.1:15432/go_template_test?sslmode=disable"
+  Driver: "postgres"
+  MaxOpenConn: 10
+  MaxIdleConn: 10
+
+Migrator:
+  DSN: "postgres://go_template:root@127.0.0.1:15432/go_template_test?sslmode=disable"
```

Comment:
hi @cymon1997 can we use the same variable as `DB.DSN` ? wdyt

Reply:
nevermind, will remove the base DB and re-add with the base db client repository instead. thanks!

Reply:
okay sure